app/assets/stylesheets/task.scss

```diff
@@ -38,6 +38,16 @@
   color: #92977E;
 }
 
+.tag-individual {
+  margin-bottom: 1rem;
```

Comment:
Properties should be ordered border-radius, font-size, height, line-height, margin-bottom, padding, vertical-align